addons/binding/org.openhab.binding.camera/ESH-INF/config/cameraConfig.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+    <config-description uri="thing-type:camera:camera">
+        <parameter name="poll" unit="s" type="integer">
+            <label>Poll time:</label>
```

Comment:
I doubt whether the colon should be included here, please validate in other bindings.

Reply:
OK